packages/components/psammead-headings/src/index.jsx

```diff
@@ -45,3 +59,19 @@ SubHeading.propTypes = {
   script: shape(scriptPropType).isRequired,
   service: string.isRequired,
 };
+
+export const PageHeading = styled.h1.attrs(() => ({
+  id: 'content',
+}))`
+  color: ${C_METAL};
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansRegular(service)};
+  margin: 0;
+  padding: ${GEL_SPACING_DBL} 0 ${GEL_SPACING_TRPL};
```

Comment:
What are people's thoughts on adding the spacing directly in the component?

When we generalized the Story and Bulletin promos and the Media Indicator, we tried to remove any outer spacing and apply this in Simorgh. 

The designs seem to have the same padding for IDX and Most Read page, but this component could be used in a different page type which could require a different spacing cc @sareh

Reply:
I’m wondering if we should add the padding logic to our HeadingsContainer in simorgh, and export it from there, this way our psammead comp is generalised and if we do need to update the spacing in the future it would quicker to do if it’s in simorgh.